ports/bddisasm/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_from_github(

```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm")

vcpkg_from_github(
```
Seems it doesn't support arm, it would be better to add the unsupported message here.